src/SDKs/dirs.proj

```diff
@@ -36,6 +36,8 @@
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Relay\Management.Relay\Microsoft.Azure.Management.Relay.csproj" />
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Resource\Management.ResourceManager\Microsoft.Azure.Management.ResourceManager.csproj" />
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Scheduler\Management.Scheduler\Microsoft.Azure.Management.Scheduler.csproj" />
+		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Search\Management\Management.Search\Management.Search.csproj" />
```

Comment:
@brjohnstmsft these files are no longer needed.
These are kept as a backup incase we need to switch back to declarative model for building projects.
But as of today, no changes are required for this file, but will not break anything if you make any updates.
Do not revert, just as an FYI